instances/technique/eyeMovementTracking.jsonld

```diff
@@ -4,11 +4,14 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/eyeMovementTracking",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
+  "definition": "'Eye movement tracking' is a technique to measure the eye movement and/or position to determine what an individual was focusing on at a given time.",
   "description": null,
   "interlexIdentifier": null,
```

Comment:
@tgbugs would this reference be correct? http://uri.interlex.org/base/ilx_0417680

Reply:
I added it, it sound correct to me. We can also fix it later in case it is wrong.